Makefile

```diff
@@ -124,14 +124,15 @@ local-git: ## Setup in-cluster git repository
 ###################
 # Install and Run #
 ###################
-.PHONY: install install-manifests wait run cert-mgr uninstall
+.PHONY: install install-manifests wait run debug debug-helper cert-mgr uninstall
 
 ##@ install/run:
 
 install: generate cert-mgr install-manifests wait ## Install rukpak
 
+MANIFESTS_DIR ?= manifests
 install-manifests:
-	$(KUBECTL) apply -k manifests
+	$(KUBECTL) apply -k $(MANIFESTS_DIR)
```

Comment:
doesn't this mean that even for normal (non debug) install that you're going to get the debug manifests?

Reply:
By default the manifests dir is set to the old value, `manifests` on line 133, so unless the `debug` target is used nothing from the debug manifests should be included with the apply command.